### PR TITLE
fix(DB/DustwallowMarsh): add pathing to Garn Mathers

### PR DESCRIPTION
**Pull request by elthehablo**
## Changes Proposed:
-  add path to Garn Mathers
-  don't change loot, as it is unconfirmed that it is wrong

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9721 
- Closes https://github.com/chromiecraft/chromiecraft/issues/2582

## SOURCE:

Sniffed waypoints from [Bench](https://github.com/heyitsbench)

## Tests Performed:

- tested

## How to Test the Changes:

Go to Dustwallow Marsh and check that he now has a path

## Known Issues and TODO List:

- [ ] check loot issue

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
For the record, he did not drop anything on Wrath Classic, though checking for a possible retail Wrath recording of his loot would solidify my opinion on that.

**Comment by elthehablo** [comment]
> For the record, he did not drop anything on Wrath Classic, though checking for a possible retail Wrath recording of his loot would solidify my opinion on that.

Inclined to believe you

File: data/sql/updates/pending_db_world/garn_mathers_path.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+(@NPC, 11, -3000.9695, -4383.346, 10.6955185, 0),
+(@NPC, 12, -3011.9897, -4373.612, 9.795075, 0),
+(@NPC, 13, -3023.2007, -4366.656, 10.3360815, 0),
+(@NPC, 14, -3026.9507, -4364.906, 10.0860815, 0),
+(@NPC, 15, -3040.5425, -4358.04, 8.381775, 0),
+(@NPC, 16, -3048.0571, -4343.4937, 8.053871, 0);
+
+DELETE FROM `creature` WHERE `id1` = 23679 AND `guid` = 30758;
+INSERT INTO `creature` (`guid`, `id1`, `id2`, `id3`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(30758, 23679, 0, 0, 1, 0, 0, 1, 1, 1, -3050.214, -4329.639, 8.156482, 1.730132341384887695, 180, 0, 0, 1536, 0, 2, 0, 0, 0, '', 0);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/garn_mathers_path.sql:24]: New creature or just updating a few columns? Add the VerifiedBuild as well if it uses all new values

File: data/sql/updates/pending_db_world/garn_mathers_path.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+DELETE FROM `creature_addon` WHERE `guid` = 30758;
+INSERT INTO `creature_addon` (`guid`, `path_id`) VALUES (30758, @NPC);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/garn_mathers_path.sql:27]: bytes2 = 1 for equipment to show

**Comment by Gultask** [comment]
Change EOL conversion to Unix
image

File: data/sql/updates/pending_db_world/garn_mathers_path.sql (hunk 1 above)
Comment (elthehablo) [reply data/sql/updates/pending_db_world/garn_mathers_path.sql:24]: Updating posis and changing movementtype (path)

**Comment by heyitsbench** [comment]
Don't forget [CC issue](https://github.com/chromiecraft/chromiecraft/issues/2582).

**Review by Gultask — APPROVED** [review]

**Pull request merged**